lib/components/UploadList.jsx

```diff
@@ -69,13 +69,19 @@ var UploadList = React.createClass({
     var showDetailsThisUpload = _.includes(this.state.showErrorDetails, upload.key);
     var errorDetails = showDetailsThisUpload ? (<div className="UploadList-error-details">{upload.error.debug}</div>) : null;
     var showErrorsText = showDetailsThisUpload ? this.props.text.HIDE_ERROR : this.props.text.SHOW_ERROR;
-    
+    var errorMessage = upload.error.driverLink ? <div className="UploadList-error-message-wrapper">
```

Comment:
Is there a way I can try out the `driverLink` thing or is that impossible on my system w/pre-El Capitan OS X?